scipy/stats/stats.py

```diff
@@ -3361,8 +3363,8 @@ def ttest_ind_from_stats(mean1, std1, nobs1, mean2, std2, nobs2,
                                              std2**2, nobs2)
     return _ttest_ind_from_stats(mean1, mean2, denom, df)
 
-
-def ttest_ind(a, b, axis=0, equal_var=True):
+    
+def ttest_ind(a, b, axis=0, equal_var=True, permutations=1):
```

Comment:
OK. @dengemann just to clarify, did you mean change the permutation parameter to default to None?